src/pip/_internal/resolution/resolvelib/base.py

```diff
@@ -143,8 +143,8 @@ def source_link(self):
         # type: () -> Optional[Link]
         raise NotImplementedError("Override in subclass")
 
-    def iter_dependencies(self, with_requires):
-        # type: (bool) -> Iterable[Optional[Requirement]]
+    def iter_dependencies(self, with_requires, ignore_requires_python):
```

Comment:
```suggestion
    def iter_dependencies(self, with_requires, with_requires_python):
```

For consistency internally in the `resolution.resolvelib` context?